File: 백준/Bronze/2480. 주사위 세개/주사위 세개.py
dice = []
count = 0
eye_count = [0] * 7
money = 0

dice += input().split()
dice = list(map(int, dice))
dice.sort()

for i in dice:
    eye_count[i] += 1
    

for i in range(len(eye_count)):
    if(eye_count[i] == 3):
        money = 10000 + i*1000
        break
    elif(eye_count[i] == 2):
        money = 1000 + i*100
        break
    else:
        money = dice[2] * 100

print(money)

File: 백준/Bronze/2480. 주사위 세개/주사위 세개.java
import java.util.Arrays;
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int[] dice = new int[3];
		int[] eyeCount = new int[7]; 
		int money = 0;
		
		
		for (int i = 0; i < dice.length; i++) {
			dice[i] = sc.nextInt();
		}
		
		Arrays.sort(dice);
		
		for (int i : dice) {
			eyeCount[i]++;
		}
		
		for (int i = 0; i < eyeCount.length; i++) {
			if(eyeCount[i] == 3) {
				money = 10000+i *1000;
				break;
			}
			else if(eyeCount[i] == 2) {
				money = 1000+i *100;
				break;
			}
			else {
				money = dice[2] * 100;
			}
		}
		System.out.println(money);
		
	}
	

}
